frontend/src/MakerApp.tsx

```diff
@@ -37,9 +39,14 @@ export default function App() {
     const priceInfo = useLatestEvent<PriceInfo>(source, "quote");
 
     const toast = useToast();
-    let [minQuantity, setMinQuantity] = useState<string>("100");
-    let [maxQuantity, setMaxQuantity] = useState<string>("1000");
-    let [orderPrice, setOrderPrice] = useState<string>("10000");
+    let [minQuantity, setMinQuantity] = useState<string>("10");
+    let [maxQuantity, setMaxQuantity] = useState<string>("100");
+    let [orderPrice, setOrderPrice] = useState<string>("0");
+
+    const spread = 1.03;
+    if (priceInfo && orderPrice === "0") {
+        setOrderPrice((priceInfo.bid * spread).toString());
```

Comment:
He should be using the `ask` price. 

The `ask` price is the `lowest` price the seller is willing to sell the price. 
In our case, the `maker` is `selling` to the `taker`.

Maybe it's clearer if you look at the values we get from BitMEX. `ask>bid`: you buy (pay the `ask` price) at a higher value then you sell (pay the `bid` price).

Reply:
Yeah, but given an arbitrage model, the seller wants to be covered for the bid (because that's what the seller goes to market with. Or am I missing something?

Reply:
I think my thinking was just wrong :)